### ISSUE TITLE
Ability to remove the blank comparator options from filters

### ISSUE THREAD

**Issue by sean-ww** [AllenFang/react-bootstrap-table#1248]
When setting a default comparator, in many cases, the blank comparator option will no longer be required. Could the blank comparator option be made optional?
e.g.
```
filter={ { 
    type: 'NumberFilter', 
    numberComparators: [ '=', '>', '<=' ],
    defaultValue: { number: 50, comparator: '>' },
    forceSelect: true
} }
```
With forceSelect removing the blank comparator option.

**Comment by AllenFang** [comment]
@sean-ww, I'll consider to support this feature if there's no any critical problem on implement this feature, thanks

**Comment by sean-ww** [comment]
Thanks @AllenFang. If it defaults to false, it shouldn't cause any issues.

**Comment by AllenFang** [comment]
Fixed on `v3.2.2` and check https://github.com/AllenFang/react-bootstrap-table/blob/master/examples/js/column-filter/number-filter-without-empty-option.js#L32.  Thanks
